src/plugins/intel_gpu/docs/simplified_workflow.md

```diff
@@ -0,0 +1,12 @@
+# GPU plugin workflow
+
+The simplified workflow in the GPU plugin is shown on the picture below (click on image for higher resolution):
+
+![](../../../../docs/img/ov_intel_gpu_plugin_diagram.png)
```

Comment:
Why do you need this file in common docs folder?
Please create internal `img` folder in this component

Reply:
Can we change to mermaid?

Reply:
The version in mermaid would be way too lacking as not all of the elements can be recreated with it. For instance, classDiagram  which would need to be implemented here because of the relation types such as aggregation and composition cannot use inline styling which would be required to recreate such elements as "User App", "OV Core", "Intel GPU plugin". Currently it is not possible to style the classDiagram elements, and I quote: "Due to limitations with existing markup for class diagrams, it is not currently possible to define css classes within the diagram itself." (For more information, see: https://mermaid.js.org/syntax/classDiagram.html#styling). Therefore, I do believe the use of mermaid in this and similar cases does not pay off, as it has limited capabilities.